website/blog/2020/11.20-Case-Study-Migrating-ETCD-Volumes-in-Production.md

```diff
@@ -35,7 +35,7 @@ We from metal-stack learned quite a lot about the capabilities of Gardener throu
 
 ## How Gardener Manages ETCDs
 
-In our infrastructure, we use the Gardener to provision Kubernetes clusters on bare metal machines in our own data centers using <span style="white-space:nowrap">[metal-stack](https://metal-stack.io/)</span>. Even if the entire stack could be running on-premise, our initial [seed cluster](https://github.com/gardener/gardener/blob/master/docs/concepts/apiserver.md#seeds) and the [metal control plane](https://docs.metal-stack.io/stable/overview/architecture/#Metal-Control-Plane) are hosted on GKE. This way, we do not need to manage a single Kubernetes cluster in our entire landscape manually. As soon as we have Gardener deployed on this initial cluster, we can spin up further Seeds in our own data centers through the concept of [shooted seeds](https://github.com/gardener/gardener/blob/master/docs/usage/shooted_seed.md).
+In our infrastructure, we use the Gardener to provision Kubernetes clusters on bare metal machines in our own data centers using <span style="white-space:nowrap">[metal-stack](https://metal-stack.io/)</span>. Even if the entire stack could be running on-premise, our initial [seed cluster](https://github.com/gardener/gardener/blob/master/docs/concepts/apiserver.md#seeds) and the [metal control plane](https://docs.metal-stack.io/stable/overview/architecture/#Metal-Control-Plane) are hosted on GKE. This way, we do not need to manage a single Kubernetes cluster in our entire landscape manually. As soon as we have Gardener deployed on this initial cluster, we can spin up further Seeds in our own data centers through the concept of ManagedSeeds.
```

Comment:
Why are we removing te link from the ManagedSeeds? 
https://gardener.cloud/docs/gardener/usage/managed_seed/